src/app/client/src/app/modules/groups/components/activity/activity-list/activity-list.component.html

```diff
@@ -38,17 +38,21 @@
 <app-add-activity *ngIf="!showLoader && ((activityList | json) === '{}')"></app-add-activity>
 
 <!-- Remove Activity from the group -->
-<sui-modal [mustScroll]="true" [isClosable]="true" [transitionDuration]="0" [size]="'small'" class="sb-modal sb-error"
-appBodyScroll (dismissed)="toggleModal()" #modal *ngIf="showModal">
-<div class="sb-modal-header">
-  {{resourceService?.frmelmnts?.lbl?.removeActivity}} ?
-</div>
-<div class="sb-modal-content">
-  <p> {{resourceService?.messages?.imsg?.removeActivityWarning}}</p>
-</div>
-<div class="sb-modal-actions">
-  <button class="sb-btn sb-btn-normal sb-btn-error" tabindex="0"
-    (click)="removeActivity()">{{resourceService?.frmelmnts?.btn?.remove}}</button>
-</div>
-
-</sui-modal>
+<app-modal-wrapper *ngIf="showModal" #modal [config]="{disableClose: true, size: 'small'}">
```

Comment:
@ArtiSingh23 @VenkatKadiveti What about (dismissed)="toggleModal()" option. Is that use-case is working fine?